api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -1100,6 +1100,12 @@ public static String getOpenmrsProperty(String property) {
 	 * @since 1.11
 	 */
 	public static final Integer SEARCH_INDEX_VERSION = 3;
+
+	/**
+	 * @since 1.12
+	 * Specifies the uuid of concept which represents drug non coded
+	 */
+	public static final String GP_DRUG_NON_CODED_CONCEPT_UUID = "order.drugNonCodedConceptUuid";
```

Comment:
According to the ticket description, I thought the GP name is supposed to be drugOrder.drugNonCoded, i actually think drugOrder.drugNonCodedConceptUuid